web_app/css/styles.css

```diff
@@ -752,6 +752,11 @@ body {
     font-family: 'Inter', sans-serif;
     text-transform: uppercase;
 }
+
+input[type=time]::-webkit-datetime-edit-ampm-field {
```

Comment:
As I understood it won't work because it will hide time PM/AM for people with enlish locale but won't remove this field from input validation. Therefore, the validation should be somehow overriden for users with english locale and the AM or PM should be just ingored and not required